ui/src/cloud/components/GetAssetLimits.tsx

```diff
@@ -0,0 +1,57 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+
+// Actions
+import {getAssetLimits as getAssetLimitsAction} from 'src/cloud/actions/limits'
+
+// Components
+import {
+  TechnoSpinner,
+  SpinnerContainer,
+  RemoteDataState,
+} from '@influxdata/clockface'
+import {AppState} from 'src/types'
+
+interface StateProps {
+  status: RemoteDataState
+}
+
+interface DispatchProps {
+  getAssetLimits: typeof getAssetLimitsAction
+}
+
+type Props = StateProps & DispatchProps
+
+class LimitChecker extends PureComponent<Props> {
```

Comment:
Seems to be a different name than the file name.

```suggestion
class GetAssetLimits extends PureComponent<Props> {
```